DataScience/loggers/logger_wrapper.py

```diff
@@ -0,0 +1,41 @@
+from fluent import asyncsender as sender
+import json
+import sys
+import traceback
+from loggers.geneva_logger import GenevaLogger 
+from loggers.console_logger import ConsoleLogger
+
+class Logger:
+    loggers = []
+
+    @classmethod
+    def create_loggers(cls, geneva=False, **geneva_args):
+        Logger.loggers.append(ConsoleLogger())
+        if geneva:
+            cls.loggers.append(GenevaLogger(**geneva_args))
```

Comment:
Inconsistency between Logger.loggers and cls.loggers.

Reply:
good catch, fixed